core-web/src/main/java/org/silverpeas/core/web/treemenu/process/TreeHandler.java

```diff
@@ -47,14 +47,18 @@ public class TreeHandler {
   private TreeHandler() {
   }
 
+  static boolean useOrder = false;
+
   /**
    * get information from request and build a level menu
    * @param request
    * @param menuType
-   * @return
+   * @param useCurrentOrder
+   * @return Json Array
    * @throws RemoteException
    */
```

Comment:
The parameters and the return value miss description.
The RemoteException isn't thrown by the method and hence it can be removed

Reply:
Done.